src/backend/main.py

```diff
@@ -1,25 +1,68 @@
 import glob
+import sys
+import os
+import argparse
+from pipeline.templates import full_pipeline, default_pipeline
+from pipeline.mat import Mat
+from google.cloud import storage
 from pipeline.templates import full_pipeline, default_pipeline, training_pipeline, nutrient_pipeline
 import firebase_admin
 from firebase_admin import credentials, firestore
 import asyncio
 from pipeline.mat import Mat, Channels
+from pipeline.config import CloudConfig
 import numpy as np
 
 def main():
     """Main entry point."""
 
+    # parse arguments
+    parser = argparse.ArgumentParser(description='Process cloud arguments')
+    parser.add_argument('--path', dest='path', default=None)
+    parser.add_argument('--mode', dest='mode', default=None)
```

Comment:
Can you move this to `if __name__ == "__main__":`? That's the practice afaik since you separate the program input from the main logic.